### PR TITLE
Import DB further speed improvements

### PR DESCRIPTION
**Pull request by iulianpascalau**
- altered some config storage values in import-db mode to speed-up storage syncing

### CODE REVIEW

File: cmd/node/main.go [hunk 1]
```diff
@@ -1530,9 +1530,27 @@ func applyCompatibleConfigs(isInImportMode bool, importDbNoSigCheckFlag bool, lo
 		config.Heartbeat.DurationToConsiderUnresponsiveInSec = math.MaxInt32
 		config.Heartbeat.MinTimeToWaitBetweenBroadcastsInSec = math.MaxInt32 - 2
 		config.Heartbeat.MaxTimeToWaitBetweenBroadcastsInSec = math.MaxInt32 - 1
+
+		changeStorageConfigs(config)
 	}
 }
 
+func changeStorageConfigs(config *config.Config) {
```
Comment (bogdan-rosianu) [inline cmd/node/main.go:1538]: `alterStorageConfigsForDBImport` ?
also, you could send value instead of pointer

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]

File: cmd/node/main.go (hunk 1 above)
Comment (iulianpascalau) [reply cmd/node/main.go:1538]: I will change the name but I need to keep the pointer to be modified by the function

Comment (iulianpascalau) [reply cmd/node/main.go]: done

**Review by bogdan-rosianu — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Review by sasurobert — DISMISSED** [review]

File: config/config.go [hunk 2]
```diff
@@ -234,7 +234,7 @@ type StateTriesConfig struct {
 type TrieStorageManagerConfig struct {
 	PruningBufferLen   uint32
 	SnapshotsBufferLen uint32
-	MaxSnapshots       uint8
+	MaxSnapshots       uint32
```
Comment (bogdan-rosianu) [inline config/config.go:237]: could this cause comapatibility issues?

**Review by bogdan-rosianu — DISMISSED** [review]

File: config/config.go (hunk 2 above)
Comment (iulianpascalau) [reply config/config.go:237]: No, this is strictly a processing component. The max snapshot value is strictly used when deciding to keep (or not) an older trie snapshot. This is saved nowhere.

**Review by LucianMincu — APPROVED** [review]
conflicts

**Review by sasurobert — APPROVED** [review]

**Review by stefanandy — APPROVED** [review]
System test passed. succesfull import
--doubled checked with developer--

**Review by raduchis — APPROVED** [review]

**Pull request merged**